automation_tools/__init__.py

```diff
@@ -1622,6 +1623,30 @@ def setenforce(mode):
     run('setenforce {0}'.format(mode))
 
 
+def host_pings(host):
+    """This ensures the given IP/hostname pings succesfully.
+
+    :param host: A string. The IP or hostname of host.
+
+    """
+    if host is None:
+        print("The hostname/IP is not provided to ping. \
+        Terminating the execution !!")
+        sys.exit(1)
+    while True:
+        command = subprocess.Popen(
+            'ping -c1 {0}'.format(host),
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            shell=True
+        )
+        output = command.communicate()[0]
+        if '1 received' in output:
```

Comment:
Yeah, Already working on it. Return codes are good to verify. Currently working on timeout thing.